### ISSUE TITLE
Markup and a11y when defining a list of chip groups

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3060]
From a conversation with @jessiehuff and @jgiardino, let's review our use of a group of chip groups and ensure we're using the structure below.

We discussed that a list of chip groups should be defined by the thing that creates the list (ie, a toolbar group that has chip groups as the group items). The key takeaways are that the parent should be a `ul[role="list"]`, with an `li[role="listitem"]` for each item that contains a chip group. The redundant roles are needed for screen readers to announce the lists properly.

```
ul.some-group[role="list"]
  li.some-group-list-item[role="listitem"]
    // below is a standard chip group with no modifications
    div.chip-group
      span.chip-group-title[aria-hidden="true"][id="foo"]
      ul.chip-group-list[aria-labelledby="foo"]
        li.chip-group-list-item
           div.chip
  li.some-group-list-item[role="listitem"]
    // below is a standard chip group with no modifications
    div.chip-group
      span.chip-group-title[aria-hidden="true"][id="foo"]
      ul.chip-group-list[aria-labelledby="foo"]
        li.chip-group-list-item
           div.chip
```
